app/utils/types.ts

```diff
@@ -0,0 +1,86 @@
+export type AggregatorInputParams = {
+    quoteID: string,
+    defaultAsset: string,
+    defaultNetwork?: string,
+    defaultPaymentMethod: string,
+
+    fiatCurrency: string,
+    presentFiatAmount: string,
+}
+
+export type GenerateSecureTokenRequest = {
```

Comment:
Should there be a response object type too?

Reply:
the response is simply a string containing the token so its not necessary